test/mocks/objects/notify-link-to-file.js

```diff
@@ -10,7 +10,20 @@ const confirmEmail = {
   confirm_email_before_download: true
 }
 
+const retentionPeriodSet = {
+  ...defaultObject,
+  retention_period: '78 weeks'
+}
+
+const confirmEmailAndRetentionPeriodSet = {
```

Comment:
Final point: let's use the `confirmEmail` and `retentionPeriodSet` objects instead of repeating setting them to particular values, i.e.
```

const confirmEmailAndRetentionPeriodSet = {
   ...defaultObject,
   ...confirmEmail,
   ...retentionPeriodSet
}

```

Reply:
And let's update the file `test/mocks/objects/notifications-node-client.js` to use this combo object `confirmEmailAndRetentionPeriodSet`

Reply:
Already updated : https://github.com/DEFRA/ffc-pay-statement-publisher/blob/122793_Set_default_retention_period_on_Notify_files_to_18_months/test/mocks/objects/notifications-node-client.js

Reply:
https://github.com/DEFRA/ffc-pay-statement-publisher/blob/122793_Set_default_retention_period_on_Notify_files_to_18_months/test/mocks/objects/notify-link-to-file.js

line 18